src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorFrontendEndpointObject.cs

```diff
@@ -107,15 +108,16 @@ public override void ExecuteCmdlet()
             {
                 Name = Name,
                 HostName = HostName,
-                SessionAffinityEnabledState = !this.IsParameterBound(c => c.SessionAffinityEnabledState)? PSEnabledState.Disabled : SessionAffinityEnabledState,
+                SessionAffinityEnabledState = !this.IsParameterBound(c => c.SessionAffinityEnabledState) ? PSEnabledState.Disabled : SessionAffinityEnabledState,
                 SessionAffinityTtlSeconds = !this.IsParameterBound(c => c.SessionAffinityTtlInSeconds) ? 0 : SessionAffinityTtlInSeconds,
                 WebApplicationFirewallPolicyLink = WebApplicationFirewallPolicyLink,
                 CertificateSource = CertificateSource,
                 CertificateType = CertificateType,
                 Vault = Vault,
                 SecretName = SecretName,
                 SecretVersion = SecretVersion,
-                ProtocolType = ProtocolType
+                ProtocolType = this.IsParameterBound(c => c.ProtocolType) ? ProtocolType : "ServerNameIndication",
+                MinimumTlsVersion = !this.IsParameterBound(c => c.SessionAffinityTtlInSeconds) ? "1.2" : MinimumTlsVersion
```

Comment:
>SessionAffinityTtlInSeconds [](start = 66, length = 27)

mintlsversion

Reply:
The Swagger refers to it as MinimumTlsVersion. ?

Reply:
I am refreing to "!this.IsParameterBound(c => c.SessionAffinityTtlInSeconds)". It shouldnt be SessionAffinityTtlInSeconds

---
In reply to: [331267175](https://github.com/jessicl-ms/azure-powershell/pull/1#discussion_r331267175) [](ancestors = 331267175)

Reply:
I am refreing to "!this.IsParameterBound(c => c.SessionAffinityTtlInSeconds)". It shouldnt be SessionAffinityTtlInSeconds

---
In reply to: [330856087](https://github.com/jessicl-ms/azure-powershell/pull/1#discussion_r330856087) [](ancestors = 330856087)